pkg/bpf/bpf.go

```diff
@@ -204,21 +204,33 @@ func (l *BpfLoader) Stop() {
 
 func NewVersionMap(config *options.BpfConfig) *ebpf.Map {
 	var versionPath string
+	var versionMap *ebpf.Map
 	if config.AdsEnabled() {
-		versionPath = filepath.Join(config.BpfFsPath + "/bpf_kmesh/map/")
+		versionPath = filepath.Join(config.BpfFsPath + "/bpf_kmesh/map/kmesh_version")
 	} else if config.WdsEnabled() {
-		versionPath = filepath.Join(config.BpfFsPath + "/bpf_kmesh_workload/map/")
+		versionPath = filepath.Join(config.BpfFsPath + "/bpf_kmesh_workload/map/kmesh_version")
 	}
 
 	_, err := os.Stat(versionPath)
 	if err == nil {
-		m := recoverVersionMap(config, versionPath)
-		if m != nil {
-			SetStartStatus(m)
-			return m
+		versionMap = recoverVersionMap(config, versionPath)
```

Comment:
Don't you handle errors when they occur?

Reply:
Only the successful recovery from the pin directory needs to be processed. In other cases, it will be started in the Normal state.